test-cases/sct-cases/dns-cluster-5min.yaml

```diff
@@ -0,0 +1,16 @@
+# TODO: need to qualify
```

Comment:
With regards to our conversation about jobs, this one also need a job.

Reply:
add a pipeline for it